### DISCUSSION TITLE
Widget: STYLE BLOCK <-- What does it do?

### DISCUSSION THREAD

**Discussion by aykaramba** [rabbibotton/clog#132] [Q&A] [answered]
HI.

What does the Style Block widget do?

The manual states:  Ideally style blocks should be created in the (head body) clog-element but can be placed anywhere on a document and are applied as found in the document. Although they are not 'scoped'. Media is a css media query defaulting to all. To load CSS style sheets from files see [LOAD-CSS](file:///home/rtaylor/common-lisp/clog/doc/clog-manual.html#CLOG:LOAD-CSS%20GENERIC-FUNCTION) in clog-document. The add-style method can be used or can directly use the [TEXT](file:///home/rtaylor/common-lisp/clog/doc/clog-manual.html#CLOG:TEXT%20GENERIC-FUNCTION) method to access blocks content.

So I guess we would put CSS in the content box and all the child widgets would inherit the style?  What is the envisioned use case for this widget?

Thanks.

**Comment by rabbibotton** [comment, marked as answer]
Any block that is paced on the page will affect every element on the page. Sadly in html style blocks have no scope.

**Comment by aykaramba** [comment]
Thank you.
